ow_power_system/src/PowerSystemNode.cpp

```diff
@@ -382,6 +393,80 @@ void PowerSystemNode::initAndRun()
   spinner.stop();
 }
 
+/*
+ * Used if the power system is disabled. Simply publishes static values
+ * without using GSAP prognosers or reacting to faults. Mechanical power is
+ * still calculated using jointStatesCb for publication.
+ */
+void PowerSystemNode::runDisabledLoop()
+{
+  // This rate object is used to sync the cycles up to the provided Hz.
+  // NOTE: If the cycle takes longer than the provided Hz, nothing bad will
+  //       necessarily occur, but the simulation will be out of sync with real
+  //       time. Should essentially never happen with the disabled version of
+  //       the loop.
+  ros::Rate rate(m_loop_rate_hz);
+
+  // Start the asynchronous spinner, which will call jointStatesCb as the
+  // joint_states topic is published (50Hz).
+  ros::AsyncSpinner spinner(m_spinner_threads);
+
+  try
+  {
+    spinner.start();
+  }
+  catch(const std::runtime_error& e)
+  {
+    ROS_ERROR_STREAM("OW_POWER_SYSTEM ERROR: Encountered a std::runtime_error"
+                    << " while starting up asynchronous ROS spinner threads!");
+    return;
+  }
+
+  bool skip_first_loop = true;
+
+  while (ros::ok())
+  {
+    // Create the published messages and set them to their ideal states.
+    owl_msgs::BatteryRemainingUsefulLife rul_msg;
+    owl_msgs::BatteryStateOfCharge soc_msg;
+    owl_msgs::BatteryTemperature tmp_msg;
+
+    rul_msg.value = m_max_horizon_secs;   // Will always be the max possible RUL
+    soc_msg.value = 1.0;                  // Hardcoded 100% battery charge
+    tmp_msg.value = 20;                   // Hardcoded 20C battery temperature
```

Comment:
Can `initial_soc` and `initial_temperature` from `system.cfg` be easily used to avoid the hardcoding?